docs/getting-started/air-gapped-environment.md

```diff
@@ -0,0 +1,121 @@
+---
+sidebar_position: 3
+---
+
+# Air-gapped environment
+
+Rancher Turtles provides support for an air-gapped environment out-of-the-box by leveraging features of the Cluster API Operator, the required dependency for installing Rancher Turtles.
+
+To provision and configure Cluster API providers, Turtles uses the **CAPIProvider** resource to allow managing Cluster API Operator manifests in a declarative way. Every field provided by the upstream CAPI Operator resource for the desired `spec.type` is also available in the `spec` of the **CAPIProvider** resouce.
+
+To install Cluster API providers in an air-gapped environment the following will need to be done:
+
+1. Configure the operator for an air-gapped environment:
+   - Manually fetch and store a helm chart for the operator.
+   - Provide image overrides for the operator in from an accessible image repository.
+2. Configure providers for an air-gapped environment:
+   - Provide fetch configuration for each provider from an accessible location (e.g., an internal git repository) or from pre-created ConfigMaps within the cluster.
```

Comment:
```suggestion
   - Provide fetch configuration for each provider from an accessible location (e.g., an internal github/gitlab server) or from pre-created ConfigMaps within the cluster.
```

Reply:
we only support github and gitlab for now(same as clusterctl)

Reply:
It said GitHub originally and i changed it, my bad. Thanks @alexander-demicev  and @furkatgofurov7

Reply:
Fixed now, thanks!